application/src/domain/value_object/orderStatus.ts

```diff
@@ -21,6 +26,7 @@ export class OrderStatus {
       OrderStatus.EM_PREPARACAO,
       OrderStatus.PRONTO,
       OrderStatus.ENTREGUE,
+      OrderStatus.PENDING,
```

Comment:
Pelo que vi o status de pendente seria relacionado à pendente de preparação no endpoint de fila, mas ele não referencia diretamente um status do pedido com o mesmo nome, na verdade referencia o `AGUARDANDO_PREPARO`, poderiamos mudar ele para `PENDING` se for o caso, ou `PENDENTE` já que os demais estão em português.

Reply:
Feito